Documentation/contributing/testing/e2e.rst

```diff
@@ -137,6 +137,7 @@ The Kubernetes tests support the following Kubernetes versions:
 * 1.20
 * 1.21
 * 1.22
+* 1.23
 
 By default, the Vagrant VMs are provisioned with Kubernetes 1.21. To run with any other
```

Comment:
Should be 1.22 after `jenkinsfiles/ginkgo-kernel.Jenkinsfile` has been changed.

Reply:
Fixed locally, will repushed once tests have passed